lib/gcov/Config.uk

```diff
@@ -0,0 +1,25 @@
+menuconfig LIBGCOV
+	bool "gcov: Code coverage support"
+	default n
+	help
+		This library implements the gcov support for code coverage
+		analysis.
+
+if LIBGCOV
+
+choice LIBGCOV_OUTPUT
+	prompt "Coverage data output"
+	default LIBGCOV_OUTPUT_SERIAL_HEXDUMP
+config LIBGCOV_OUTPUT_SERIAL_HEXDUMP
```

Comment:
Right now the console option outputs binary data, which cannot be captured. These need to be converted to printable characters, similarly to `embedded-gcov`. Since it uses a permissive license, you can even port the relevant code from there, if needed.